src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write your code here
+const body = document.body;
+const logo = document.querySelector('.logo');
+
+async function handlePromise(promise) {
+  try {
+    await promise;
+
+    body.insertAdjacentHTML(
+      'beforeend',
+      `
+      <div class="message">
+        Promise was resolved!
+      </div>
+    `,
+    );
+  } catch {
```

Comment:
It's a good practice to catch the error object in the catch block to understand what went wrong. Consider using `catch(error)` to handle the error appropriately.